lib/tests/modules.sh

```diff
@@ -293,7 +293,7 @@ checkConfigOutput '^"a c"$' config.result ./functionTo/merging-attrs.nix
 
 # moduleType
 checkConfigOutput '^"a b"$' config.resultFoo ./declare-variants.nix ./define-variant.nix
-checkConfigOutput '^"a y z"$' config.resultFooBar ./declare-variants.nix ./define-variant.nix
+checkConfigOutput '^"a b y z"$' config.resultFooBar ./declare-variants.nix ./define-variant.nix
```

Comment:
I didn't recognize the problem at the time.

The assertion looks for `attrNames config.variants.foo.variants.bar.attrs`.

```
    attrs.a = 1;
    variants.foo.attrs.b = 1;
    variants.bar.attrs.y = 1;
    variants.foo.variants.bar.attrs.z = 1;
    variants.foo.variants.foo.attrs.c = 3;
```

`b` occurs in the `foo` variant, so it should also occur in the `foo`+`bar` variant.